lib/getstats.js

```diff
@@ -64,6 +64,14 @@ function _getStats(peerConnection, options) {
  * @returns {Array<MediaStreamTrack>}
  */
 function getTracks(peerConnection, kind, localOrRemote) {
+  var getSendersOrReceivers = localOrRemote === 'local' ? 'getSenders' : 'getReceivers';
+  if (peerConnection[getSendersOrReceivers]) {
+    return peerConnection[getSendersOrReceivers]().map(function(senderOrReceiver) {
+      return senderOrReceiver.track;
+    }).filter(function(track) {
+      return track;
```

Comment:
@markandrus Seems like you're expecting some `track`s to be `null` here. Am I right?

Reply:
Yes, the RTCRtpSender's `track` attribute is nullable (see [here](http://w3c.github.io/webrtc-pc/#rtcrtpsender-interface)).